src/Prima/Pyxis/Message.elm

```diff
@@ -331,3 +360,51 @@ buildClassList (Config { type_, kind }) options =
         |> List.append options.classList
         |> List.append (List.map (H.flip Tuple.pair True) options.classes)
         |> Attrs.classList
+
+
+renderIconOk : Html msg
+renderIconOk =
+    Svg.node "svg"
+        [ Attrs.attribute "role" "icon"
+        , Attrs.attribute "xmlns" "http://www.w3.org/2000/svg"
+        , Attrs.attribute "viewBox" "0 0 1024 1024"
+        ]
+        [ Svg.node "title" [] [ text "ok" ]
+        , Svg.node "path" [ Attrs.attribute "d" "M275.2 502.4c0 9.6-9.6 19.2-19.2 19.2s-19.2-9.6-19.2-19.2 9.6-19.2 19.2-19.2 19.2 9.6 19.2 19.2zm614.4-6.4v192c0 48-25.6 76.8-35.2 89.6l-41.6 44.8c-28.8 28.8-67.2 32-76.8 32H499.2c-54.4 0-86.4-28.8-92.8-35.2l-32-35.2v3.2c0 35.2-28.8 64-64 64H201.6c-35.2 0-64-28.8-64-64v-336c0-35.2 28.8-64 64-64h105.6c32 0 60.8 25.6 64 57.6l128-131.2 38.4-140.8c3.2-12.8 16-32 38.4-32h67.2c19.2 0 35.2 12.8 35.2 32v230.4h108.8c12.8 0 35.2 6.4 57.6 28.8l28.8 28.8c9.6 9.6 16 22.4 16 35.2zM336 784V451.2c0-16-12.8-28.8-28.8-28.8H201.6c-16 0-28.8 12.8-28.8 28.8V784c0 16 12.8 28.8 28.8 28.8h105.6c16 0 28.8-12.8 28.8-28.8zm518.4-288c0-6.4-3.2-9.6-6.4-16l-28.8-28.8c-12.8-12.8-25.6-16-28.8-16H675.2c-16-3.2-28.8-12.8-28.8-41.6V179.2H576l-3.2 3.2-41.6 144c0 3.2-3.2 6.4-6.4 9.6L368 499.2v233.6l60.8 64c3.2 3.2 25.6 25.6 67.2 25.6h240c3.2 0 32 0 51.2-19.2l41.6-44.8c6.4-6.4 25.6-28.8 25.6-64z" ] []
+        ]
+
+
```

Comment:
Tutte queste icone dovrebbero stare in un file `src/Prima/Pyxis/Message/Icons.elm`